src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -25,8 +26,10 @@
 def load_command_table(self, _):
 
     container_services_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.v2017_07_01.operations.'
```

Comment:
This essentially changes azure to use the latest api version instead of 2017_07_01 unless we add an entry in _shared.py for the latest Profile map. Do we want to do that?

Reply:
I already have added api versions to latest profile in _shared.py